client/core/trade.go

```diff
@@ -2086,7 +2086,7 @@ func (c *Core) tick(t *trackedTrade) (assetMap, error) {
 // This method modifies match fields and MUST be called with the trackedTrade
 // mutex lock held for reads.
 func (c *Core) resendPendingRequests(t *trackedTrade) {
-	if t.isSelfGoverned() {
+	if t.isSelfGoverned() || !c.loggedIn /* we can't send `init` or `redeem` if we are not logged in */ {
```

Comment:
No need to log a message, just move on.

Reply:
log debug maybe?

Also I think `c.loginMtx` should be held.